lib/bill_forward/entities/payment_method.rb

```diff
@@ -1,27 +1,27 @@
 module BillForward
-	class PaymentMethod < MutableEntity
-  		@resource_path = BillForward::ResourcePath.new("payment-methods", "paymentMethod")
+  class PaymentMethod < MutableEntity
```

Comment:
okay. my preference is tabs (if we have the choice).
I am not sure why _some_ of the files use double-spacing.